EvolutionModelTests/DeckTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Evolution.Model;
+
+namespace EvolutionModelTests {
+
+    [TestClass]
+    public class DeckTests{
+        [TestMethod]
+        public void DeckBegintVol() {
+            var deck = new Deck();
+            var expectedCount = 129;
```

Comment:
Ook in de tests horen dit soort magic numbers niet thuis. Wederom, als je de hoeveelheid kaarten aanpast vallen je tests opeens om. Of juist niet.